src/hooks/query/use-todo-detail.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from "@tanstack/react-query";
+
+import { todoKeys } from "@/services/query-key";
+import { getTodoDetail } from "@/services/todo";
+import { TodoResponse } from "@/types/todo";
+
+export const useTodoDetail = (todoId: number | null, isEditMode: boolean) => {
+  return useQuery<TodoResponse>({
+    queryKey: todoId ? todoKeys.detail(todoId) : ["todoDetail", "empty"],
```

Comment:
할일 상세 정보를 받는데, todoId를 전달 받지 못하는 상황이 있나요?? 없다면 굳이 todoId의 타입으로 null을 넣을 필요가 없고, 삼항연산자를 쓸 일이 없을 것 같아요

```suggestion
export const useTodoDetail = (todoId: number, isEditMode: boolean) => {
  return useQuery<TodoResponse>({
    queryKey: todoKeys.detail(todoId),
```